test/linalg/symmetric.jl

```diff
@@ -166,9 +166,11 @@ end
         # mat * vec
```

Comment:
I think that this testset was moved from https://github.com/JuliaLang/julia/blob/master/test/linalg/eigen.jl, because the symmetric eigendecomposition testset is missing from that file. (there is `"non-symmetric eigen decomposition",` `"symmetric generalized eigenproblem"` and `"Non-symmetric generalized eigenproblem"`)